src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -143,9 +143,9 @@
     </DynamicEnumProperty.Metadata>
   </DynamicEnumProperty>
 
-  <DynamicEnumProperty Name="MinimumPlatformVersion"
-                DisplayName="Minimum OS version"
-                Description="Specifies the minimum runtime version of the selected operating system."
+  <DynamicEnumProperty Name="SupportedOSPlatformVersion"
```

Comment:
@dsplaisted  Does MinimumPlatformVersion exist in .NET 5?

Reply:
I don't think so.  I think that it should only be displayed or enabled in the UI when there is a target platform in the TargetFramework.  IE it shouldn't show up for `net6.0` either, but it should show up for `net6.0-windows10.0.17763`, `net6.0-android`, etc.

Reply:
There's a condition visibility below that indicates the `SupportedOSPlatformVersion` should be shown when it's a .NET 5 project or greater and when there is a value in the `TargetPlatformIdentifier`, so I think we got that covered.

Reply:
> .NET 5 project or greater

I think it should probably be checking for .NET 6 or greater, though I think a .NET 5 project might not have a `TargetPlatformIdentifier` normally, so it may be moot.

Reply:
>I think a .NET 5 project might not have a TargetPlatformIdentifier normally, so it may be moot.

`net5.0-windows` projects would be able to set a property that didn't do anything? Not a big deal, but it would be safest to check for net6.0.